sqlalchemy_solr/http.py

```diff
@@ -90,6 +89,13 @@ def create_connect_args(self, url, **kwargs):
 
             # Prepare a session with proper authorization handling.
             session = Session()
+            #session.verify property which is bydefault true so Handled here
+            session.verify = False
```

Comment:
Two points here:

1. I would suggest keeping `session.verify` to the default `True` and the parameter can alter that to 'False`, i.e.:

```
if "verify_ssl" in url.query and url.query["verify_ssl"] in [False, "False", "false"]:
  session.verify = False
 ...
```

2. According to the [Requests documentation](https://requests.readthedocs.io/en/latest/api), there are two different session attributes: [`verify`](https://requests.readthedocs.io/en/latest/api/?highlight=verify#requests.Session.verify) which is a `boolean` and [`cert`](https://requests.readthedocs.io/en/latest/api/?highlight=verify#requests.Session.cert) which represents the path to the SSL client certificate. By contrast, only `request` method accepts `verify` parameter as either a `boolean` or `string` path to CA bundle. How does that apply to the logic below?

Reply:
`
According to the [Requests documentation](https://requests.readthedocs.io/en/latest/api), there are two different session attributes: [verify](https://requests.readthedocs.io/en/latest/api/?highlight=verify#requests.Session.verify) which is a boolean and [cert](https://requests.readthedocs.io/en/latest/api/?highlight=verify#requests.Session.cert) which represents the path to the SSL client certificate. By contrast, only request method accepts verify parameter as either a boolean or string path to CA bundle. How does that apply to the logic below?
`

Actually here I have refered below condition in _solrdbapi.py : I think then should we have to change this also?

    if verify_ssl is False:
        session.verify = False
    else:
        if ca_certs is not None:
            session.verify = ca_certs
        else:
            session.verify = True

Reply:
`
I would suggest keeping session.verify to the default True and the parameter can alter that to 'False`, i.e.:
if "verify_ssl" in url.query and url.query["verify_ssl"] in [False, "False", "false"]:
  session.verify = False 
 `
 
 Are you intending to enforce certificate verification by default unless the client explicitly sets `verify_ssl` to False?